main.go

```diff
@@ -114,11 +116,15 @@ func main() {
 		cloud.VpcID(), cloud.EC2(), mgr.GetClient(), controllerCFG.DefaultTags, ctrl.Log.WithName("backend-sg-provider"))
 	sgResolver := networking.NewDefaultSecurityGroupResolver(cloud.EC2(), cloud.VpcID())
 	elbv2TaggingManager := elbv2deploy.NewDefaultTaggingManager(cloud.ELBV2(), cloud.VpcID(), controllerCFG.FeatureGates, cloud.RGT(), ctrl.Log)
+	targetGroupMetric, err := metrics.RegisterTargetGroupInfoMetric(k8smetric.Registry)
+	if err != nil {
+		setupLog.Error(err, "failed to register target binding information metric")
```

Comment:
nit: should it be `target group` instead of `target binding`?

Reply:
Well, that depends if look at it from the AWS or k8s perspectives. The k8s resource created by alb controller for a AWS target group is a targetgroupbinding. So either would be correct...